src/Components/ADT3DSceneBuilder/ADT3DSceneBuilder.tsx

```diff
@@ -315,6 +317,16 @@ const ADT3DSceneBuilderBase: React.FC<IADT3DSceneBuilderCardProps> = (
         });
     }, []);
 
+    const setVisualRuleFormMode = useCallback(
```

Comment:
it does not seem like it needs useCallback hook here

Reply:
Ahh I see it is used as callback and you are trying to prevent recreation in every render

Reply:
Then can we rename this to handleOnActionButtonClick or something to give us the sense that this is a callback method?

Reply:
I'm following the same pattern as other callbacks in that file: 
![image](https://user-images.githubusercontent.com/55852250/194620875-b4769bc9-2b1c-4955-a3b5-82bc1559937f.png)

Trying to match the name to the dispatch action they are calling.

Reply:
I personally don't really think we need to match the names to the actions they perform. I'd rather they have a meaningful name in the context they are used. Though I do pingpong between `onClickCancel` and `clearForm` (the trigger vs the result)

Reply:
Should all the callbacks in the file then change?